lua/starfall/libs_sh/hook.lua

```diff
@@ -437,6 +437,23 @@ add("EntityFireBullets", nil, function(instance, ent, data)
 	return true, { instance.WrapObject(ent), SF.StructWrapper(instance, data, "Bullet") }
 end)
 
+-- Register privileges
+registerprivilege("entities.canTool", "CanTool", "Whether or not the user can use the toolgun on the entity", { entities = {} })
```

Comment:
This should be deleted since its already registered elsewhere.

Reply:
Allright, will do.